resources/user-guides/launch-data-elements.md

```diff
@@ -0,0 +1,86 @@
+# Data Elements
+
+Data elements are the building blocks for your data dictionary and are used to collect, organize, and deliver data across marketing and ad technology.
+
+A data element is a variable where the value can be mapped to data within the Experience Platform Mobile SDK such as visitor ID, device name, user profile, number of launches, et cetera. In Experience Platform Launch you can reference this value by its variable name. This collection of data elements becomes the dictionary of defined data that you can use to build rules for your application, and this dictionary is shared across Experience Platform Launch where it can be used with any extension added to your property.
+
+Use data elements as widely as possible throughout rule creation to consolidate the definition of dynamic data. You define data elements once, and use them in multiple places. The concept of reusable data elements if very powerful and you should use them as a best practice.
```

Comment:
-Please reword the 1st sentence as follows:
"You can use data elements during rule creation to consolidate the definition of dynamic data."

- Please reword the 2nd and 3rd sentences as follows:
"After defining your data elements, you can reuse them in multiple places. 

*Tip*: We recommend reusing data elements as a best practice."

(We should never use "very" in technical documentation!)